### ISSUE TITLE
feature request: allow setting property in setter

### ISSUE THREAD

**Issue by andreas-soroko** [cezarypiatek/CSharpExtensions#19]
Hey,

it would be great if the rule would **not** be applied to the setter

```csharp
[InitOnly]
public class TestObject
{
    private string name;

    public string Name
    {
        get => name;
        set => name = $"{MyConst.PREFIX}_{value}"; // here
    }
}
```

could be a little bit tricky because of `private string name;`

**Comment by cezarypiatek** [comment]
@andreas-soroko Thanks for sharing this idea. Should be fixed in 3.0.72. Please let me know if it works as you expected.

**Comment by andreas-soroko** [comment]
Thank you, exactly want I wanted
